src/Components/Modal/Modal.stories.jsx

```diff
@@ -0,0 +1,39 @@
+import { React, useState } from 'react'
+import Modal from './Modal'
+
+export default {
+    title: 'Modal component',
+    component: Modal
+}
+
+const Template = (args) => {
+    const [show, setShow] = useState(false)
+    const onSubmit = () => { setShow(false) }
+
+    const { title, children } = args
+
+    return (
+        <div className="App">
+            <h1>Modal demo</h1>
+            <button onClick={() => setShow(true)}>Show Modal</button>
+
+            <Modal
+                show={show}
+                title={title}
+                onSubmit={onSubmit}
+                onClose={() => setShow(false)}
```

Comment:
If you have onSubmit function which does the same onClose and onSubmit, jus reuse that function here too.

Reply:
Done